scripts/main.py

```diff
@@ -301,36 +301,45 @@ def parse_arguments():
     except Exception as e:
         print_to_console("Error: \n", Fore.RED, str(e))
 
-    if not cfg.continuous_mode:
+    if not cfg.continuous_mode or next_action_count > 0:
         ### GET USER AUTHORIZATION TO EXECUTE COMMAND ###
         # Get key press: Prompt the user to press enter to continue or escape
         # to exit
-        user_input = ""
-        print_to_console(
-            "NEXT ACTION: ",
-            Fore.CYAN,
-            f"COMMAND = {Fore.CYAN}{command_name}{Style.RESET_ALL}  ARGUMENTS = {Fore.CYAN}{arguments}{Style.RESET_ALL}")
-        print(
-            f"Enter 'y' to authorise command or 'n' to exit program, or enter feedback for {ai_name}...",
-            flush=True)
-        while True:
-            console_input = input(Fore.MAGENTA + "Input:" + Style.RESET_ALL)
-            if console_input.lower() == "y":
-                user_input = "GENERATE NEXT COMMAND JSON"
-                break
-            elif console_input.lower() == "n":
-                user_input = "EXIT"
-                break
-            else:
-                user_input = console_input
-                command_name = "human_feedback"
-                break
+        if next_action_count == 0:
+            user_input = ""
+            print_to_console(
+                "NEXT ACTION: ",
+                Fore.CYAN,
+                f"COMMAND = {Fore.CYAN}{command_name}{Style.RESET_ALL}  ARGUMENTS = {Fore.CYAN}{arguments}{Style.RESET_ALL}")
+            print(
+                f"Enter 'y' to authorise command, 'y -n' to run n continuous commands, 'n' to exit program, or enter feedback for {ai_name}...",
+                flush=True)
+            while True:
+                console_input = input(Fore.MAGENTA + "Input:" + Style.RESET_ALL)
+                if console_input.lower() == "y":
+                    user_input = "GENERATE NEXT COMMAND JSON"
+                    break
+                elif console_input.lower().startswith("y -"):
+                    try:
+                        next_action_count = abs(int(console_input.split(" ")[1]))
+                        user_input = "GENERATE NEXT COMMAND JSON"
+                    except ValueError:
+                        print("Invalid input format. Please enter 'y -n' where n is the number of continuous tasks.")
+                        continue
+                    break
```

Comment:
Maybe regex with group capturing would be a better option here?
`r"y -\(d+)`

Reply:
I'm not too sure if regex is a really necessary option here? I haven't committed this change as I'm not set on it but this method works:
```python
import re
        while True:
            console_input = input(Fore.MAGENTA + "Input:" + Style.RESET_ALL)
            if console_input.lower() == "y":
                user_input = "GENERATE NEXT COMMAND JSON"
                break
            elif console_input.lower() == "n":
                user_input = "EXIT"
                break
            else:
                match = re.match(r"y -(\d+)", console_input.lower())
                if match:
                    try:
                        next_action_count = abs(int(match.group(1)))
                        user_input = "GENERATE NEXT COMMAND JSON"
                    except ValueError:
                        print("Invalid input format. Please enter 'y -n' where n is the number of continuous tasks.")
                        continue
                    break
                else:
                    user_input = console_input
                    command_name = "human_feedback"
                    break

Reply:
im not sure its the best either, i understand the command set is supposed to be flexible at this input point, but we already expect a y and if you mess that up it can break. A bit of structure at this point and a proper command syntax may be wise.

Reply:
I think thats the route this project is going to move towards, I think looking into a command rewrite to support many options, and maybe adding a `help` to give all options instead of printing all the options everytime could be useful to look into. For now at least there is the human_feedback catch, so if a command is entered wrong it'll be sent to the ai... although maybe that should be looked at too, don't want a mistype to be pushed as an instruction and then throw off the entire ai. Could easily add a prefix like a . to signify direct communication to the bot `. Try searching google first then adding x`, and if a command isn't recognized run it through text-davinci to build a list of possible command `t -5 is an unknown command, were you instead trying x or x?`